ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-fcos.yaml

```diff
@@ -15,13 +15,26 @@ images:
 promotion:
   name: "4.3"
   namespace: origin
+raw_steps:
+- rpm_image_injection_step:
+    from: src
+    to: mco-rpms
```

Comment:
this should be `machine-config-operator-rpms

Reply:
You need rpm_image_injection_step to create `oc-base-with-rpms`, then promotion needs to look like it does in `openshift-oc-master.yaml`